### ISSUE TITLE
GoB Flips UV's

### ISSUE THREAD

**Issue by Poxygon** [JoseConseco/GoB#409]
Hello! 

I want to start by saying that im unaware if this is a GoB bug but just incase I would rather report it. Currently when I export my model onto Zbrush, sculpt on top of it, export it to Substance Painter using zbrush FBX ExportImport then my UV's und up flipped upside down. I know Zbrush works on a different axis and flips my UV's upside down from the start, this is not a problem at all if I use the GoB/GoZ button on Zbrush and blender over and over again, the model will be perfectly fine and in the right axis. The problem comes in when you export it from Zbrush to be use in a different program the UV will flip since it appears GoB automatically flips the UV's for me.  

For example the attached image below show a simple cube workflow.
- I model a sub in blender and GoB onto Zbrush for detail with the UV's in the orientation that I want. 
- In Zbrush I sculpt the detail that I want
- Export the high poly model as an FBX to be textured in Substance Painter
- Import the model into substance painter, the UV are now flip vertically. 

If I had to take a guess its due to GoB automatically flipping my UV's whenever I use it. This is fine if Im only working on Zbrush and Blender but would mean I would have to flip the UV's for every single subtool in Zbrush if I wanted to export it for any other program.   Like I said im not sure if this is a problem with GoB, Zbrush itself and how it handles UV's, my own workflow, ect.  The only solution I can think off would be flipping every subtool manually, having an option to not flip the UV's in the GoB settings if possible or having so Zbrush had an option to flip all UV's when exporting but that will never happen. I hope this help in someway!
![1](https://user-images.githubusercontent.com/129979861/230155602-3c8f6e6b-b9f1-4379-a17f-37252f85e636.JPG)
![2](https://user-images.githubusercontent.com/129979861/230155606-9c2780b3-a122-47bb-a213-dd41e91bba17.JPG)
![3](https://user-images.githubusercontent.com/129979861/230155610-883e6a8c-aaa2-497f-beb6-c2c8ba441258.JPG)
![4](https://user-images.githubusercontent.com/129979861/230155613-8ee435a7-4cb3-4e11-b558-e65721b99fb8.JPG)

**Comment by kromar** [comment]
Hi, thanks for reporting this issue, im aware of some UV flipping issues that can happen between GoB and Zbrush and im working on a solution so the flipping can be customized by the user. This should allow for different workflows like yours.
Your example will be useful to test this once i get it working!

**Comment by gryzka** [comment]
Yeah. Flipped UV is a big problem on multi UDIM models. Adding on/off function would be a great feature for artists who roks on huge models. Thank you!
